compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/Icon.kt

```diff
@@ -38,14 +38,19 @@ import androidx.compose.ui.semantics.role
 import androidx.compose.ui.semantics.semantics
 
 /**
- * Icon component that draws [imageVector] using [tint], defaulting to [LocalContentColor]. For a
- * clickable icon, see [IconButton].
+ * A Material Design icon component that draws [imageVector] using [tint], with a default value
+ * of [LocalContentColor]. If [imageVector] has no intrinsic size, this component will use the
+ * recommended default size. Icon is an opinionated component designed to be used with single-color
+ * icons so that they can be tinted correctly for the component they are placed in. For multicolored
+ * icons and icons that should not be tinted, use [Color.Unspecified] for [tint]. For generic images
+ * that should not be tinted, and do not follow the recommended icon size, use the generic
+ * [androidx.compose.foundation.Image] instead. For a clickable icon, see [IconButton].
  *
  * To learn more about icons, see [Material Design icons](https://m3.material.io/styles/icons/overview)
  *
  * @param imageVector [ImageVector] to draw inside this icon
  * @param contentDescription text used by accessibility services to describe what this icon
- * represents. This should always be provided unless this icon is used for decorative purposes, and
+ * represents. This should always be gprovided unless this icon is used for decorative purposes, and
```

Comment:
It is the same in the upstream :)

https://github.com/androidx/androidx/blob/5ae97d9514623dcb014f5f1d484c536bb9f302a6/compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/Icon.kt#L53

I wouldn't change this to avoid conflicts